pkgs/by-name/ch/chuffed/package.nix

```
@@ -0,0 +1,38 @@
+{
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  lib,
+}:
+stdenv.mkDerivation {
+  pname = "chuffed";
+  version = "0.13.2";
+
+  src = fetchFromGitHub {
+    owner = "chuffed";
+    repo = "chuffed";
+    rev = "0.13.2";
+    sha256 = "sha256-D4HEcCDcJi05AL9suc7Twtf/wjpwBkLEeumGY3nNi5g=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  configurePhase = ''
+    cmake -DCMAKE_INSTALL_PREFIX=$out -B build -S .
+  '';
+
+  buildPhase = ''
+    cmake --build build -j $NIX_BUILD_CORES
+  '';
+
+  installPhase = ''
+    cmake --build build --target install
+  '';
+
+  meta = {
+    homepage = "https://github.com/chuffed/chuffed";
+    description = "Chuffed is a state of the art lazy clause solver designed from the ground up with lazy clause generation in mind.";
```

```suggestion
    description = "The Chuffed CP solver";
    longDescription = "Chuffed is a state of the art lazy clause solver designed from the ground up with lazy clause generation in mind.";
```
`description` must be short and only contain the right (`Y`) part of `X is a Y` phrase, for longer descriptions use (self-explanatory) `longDescription`